web/packages/teleport/src/services/resources/types.ts

```diff
@@ -59,10 +59,13 @@ export type Role = {
   version: string;
 };
 
-type RoleConditions = {
-  node_labels?: Record<string, string[]>;
+export type RoleConditions = {
+  node_labels?: Labels;
+  logins?: string[];
 };
 
+export type Labels = Record<string, string | string[]>;
```

Comment:
I remember we already have [`Label`](https://github.com/gravitational/teleport/blob/d2cf47f7d352edea1462baf8233fde5c4ed5643e/web/packages/teleport/src/types.ts#L144). 
Maybe we should call this one `NodeLabels` to avoid potential confusion?

Reply:
@gzdunek No, in the next PR, I'm gonna use the same type for Kubernetes labels. And it's [type Labels map[string]utils.Strings](https://github.com/gravitational/teleport/blob/18859892bd46926fbad05a57b7b1574077a339e6/api/types/role.go#L1349) on the backend, too. I was also thinking about `RoleLabels`, but this would be confusing, too, as it doesn't refer to, well, the role labels (part of the metadata).